src/main/java/org/eclipse/yasson/internal/model/JsonbAnnotatedElement.java

```diff
@@ -30,7 +39,13 @@
      * @param element Element.
      */
     public JsonbAnnotatedElement(T element) {
-        super(element.getAnnotations());
+        Map<Class<? extends Annotation>, Annotation> annotations = new HashMap<>();
```

Comment:
Lets restrict the initial capacity a bit here since there will be a lot of these objects created an usually only 0-3 elements in the map. By default HashMaps have a capacity of 16, but here I think an initial capacity of 4 should be more appropriate. Also, we can move the `new HashMap<>()` initializer up to line 32 where the field is declared.

Reply:
Makes sense, I changed size to 4 and moved initialization to the top